bot/src/main/java/com/intern/bot/Main.java

```diff
@@ -0,0 +1,21 @@
+package com.intern.bot;
+
+import com.intern.bot.service.MyBot;
+import lombok.extern.slf4j.Slf4j;
+import org.telegram.telegrambots.meta.TelegramBotsApi;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import org.telegram.telegrambots.updatesreceivers.DefaultBotSession;
+
+@Slf4j
+public class Main {
```

Comment:
Don't remove `BotApplication` ever. This is your application entry point, if you want to use spring of course. Otherwise you don't need spring.

Reply:
> Don't remove `BotApplication` ever. This is your application entry point, if you want to use spring of course. Otherwise you don't need spring.

I think we'll be work botApi. Spring is not needed in this module, is it right?

Reply:
No, we need spring in all modules.